conda_recipes/Houdini-20.5/recipe/meta.yaml

```diff
@@ -0,0 +1,40 @@
+# @Author: Derek Zavada
+# @Date:   2024-09-05 13:35:23
+# @Last Modified by:   Your name
+# @Last Modified time: 2024-09-11 16:02:25
+{% set name = "houdini" %}
+{% set version_partial = "20.5" %}
+{% set version_minor = "332" %}
+{% set version = version_partial + "." + version_minor %}
+
+{% set bucket_name = "houdinihive-bucket-deadlinecloud" %}
```

Comment:
This is using a different convention for how the archive is provided than all the other samples. For the samples to work out of the box, they need this consistency.